src/tests/mocks/src5_mocks.cairo

```diff
@@ -1,29 +1,3 @@
-#[starknet::contract]
-mod DualCaseSRC5Mock {
-    use openzeppelin::introspection::src5::SRC5Component;
-
-    component!(path: SRC5Component, storage: src5, event: SRC5Event);
-
-    #[abi(embed_v0)]
-    impl SRC5Impl = SRC5Component::SRC5Impl<ContractState>;
-    #[abi(embed_v0)]
-    impl SRC5CamelOnlyImpl = SRC5Component::SRC5CamelImpl<ContractState>;
-    impl InternalImpl = SRC5Component::InternalImpl<ContractState>;
-
-    #[storage]
-    struct Storage {
-        #[substorage(v0)]
-        src5: SRC5Component::Storage
-    }
-
-    #[event]
-    #[derive(Drop, starknet::Event)]
-    enum Event {
-        #[flat]
-        SRC5Event: SRC5Component::Event
-    }
-}
-
 #[starknet::contract]
 mod SnakeSRC5Mock {
```

Comment:
```suggestion
mod SRC5Mock {
```
it doesn't make sense to keep calling it snake

Reply:
true!